components/GroupSearchFilters/GroupSearchFilters.js

```diff
@@ -20,55 +21,64 @@ function GroupSearchFilters(props = {}) {
   }
 
   return (
-    <Box>
-      <Box mb="l">
-        <Button onClick={handleChangeClick}>Change Filters</Button>
-        <Button variant="secondary" ml="s" onClick={handleClearClick}>
+    <Box
+      display="flex"
+      flexDirection="row"
+      justifyContent="space-between"
+      alignItems="flex-end"
+      mb="l"
+    >
+      <Box display="flex">
+        <Button
+          onClick={handleChangeClick}
+          display="flex"
+          rounded={true}
+          size="s"
+          mr="s"
+        >
+          <Icon name="filter" size="14" mr={rem('8px')} />
+          <Box as="span">Filter</Box>
+        </Button>
+        {campuses.length > 0 && (
+          <FilterButton
+            label="Campus"
+            labelDetail={campuses[0]}
+            onClick={handleChangeClick}
+          />
+        )}
+        {preferences.length > 0 && (
+          <FilterButton
+            label="Group Types"
+            labelDetail={preferences.length}
+            onClick={handleChangeClick}
+          />
+        )}
+        {subPreferences.length > 0 && (
+          <FilterButton
+            label="Lineups"
+            labelDetail={subPreferences.length}
+            onClick={handleChangeClick}
+          />
+        )}
+        {days.length > 0 && (
+          <FilterButton
+            label="Meeting Days"
+            labelDetail={days.length}
+            onClick={handleChangeClick}
+          />
+        )}
+        <Button
+          variant="link"
+          rounded={true}
+          size="s"
+          onClick={handleClearClick}
+        >
           Clear
         </Button>
       </Box>
-      <Box
-        display="flex"
-        flexDirection="row"
-        justifyContent="space-between"
-        alignItems="flex-end"
-        mb="base"
-      >
-        <Box>
-          {campuses.length > 0 && (
-            <FilterButton
-              label="Campus"
-              labelDetail={campuses[0]}
-              onClick={handleChangeClick}
-            />
-          )}
-          {preferences.length > 0 && (
-            <FilterButton
-              label="Group Types"
-              labelDetail={preferences.length}
-              onClick={handleChangeClick}
-            />
-          )}
-          {subPreferences.length > 0 && (
-            <FilterButton
-              label="Lineups"
-              labelDetail={subPreferences.length}
-              onClick={handleChangeClick}
-            />
-          )}
-          {days.length > 0 && (
-            <FilterButton
-              label="Meeting Days"
-              labelDetail={days.length}
-              onClick={handleChangeClick}
-            />
-          )}
-        </Box>
-        <Box as="p" fontWeight="bold">{`${props.resultsCount} ${
-          props.resultsCount === 1 ? 'group' : 'groups'
-        }`}</Box>
-      </Box>
-
+      <Box as="p" fontWeight="bold">{`${props.resultsCount} ${
+        props.resultsCount === 1 ? 'group' : 'groups'
+      }`}</Box>
```

Comment:
TBH, I'm not sure how much value there is in the total count at the moment.
A.) It's inaccurate, and B.) it feels awkward in the UX now.

@jordanwade, (cc: @drewbarontini ) WDYT if we just remove it for now?
```suggestion
```